Brewlet/AppDelegate.swift

```diff
@@ -86,7 +86,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     
     @IBAction func update_upgrade(sender: NSMenuItem?) {
         let animation = animateIcon()
-        let command = updatesAvailable && sender != nil ? "upgrade" : "update"
+        let command = (outdatedPackageCount > 0) && sender != nil ? "upgrade" : "update"
```

Comment:
In the previous version, is `updatesAvailable` the same as `self.updatesAvailable`?

Reply:
Yes, it's the same. `self` must be used explicitly explicitly in closures.

Reply:
Gotcha, thanks for the clarification.